python/mcap/mcap/stream_reader.py

```diff
@@ -70,9 +77,13 @@ def get_chunk_data_stream(
     chunk: Chunk, validate_crc: bool = False
 ) -> Tuple[ReadDataStream, int]:
     if chunk.compression == "zstd":
+        if zstandard is None:
+            raise ImportError("zstandard is required to decompress zstd chunks")
```

Comment:
I'm not sure about the convention for exceptions of this nature, but assuming ImportError is the right choice here, I would set the [keyword arguments it takes](https://docs.python.org/3/library/exceptions.html#ImportError) here.

Reply:
In general I'd be surprised if I saw an import error at runtime. I would define a new error class for `UnsupportedCompressionError` and raise that here. I would also raise the same exception if the compression string was something other than "zstd", "lz4", or "".